src/App.scss

```diff
@@ -82,10 +89,13 @@ form {
 }
 /* Set core body defaults */
 body {
-  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
   scroll-behavior: smooth;
   text-rendering: optimizeSpeed;
   line-height: 1.5;
+  background-image: url("../src/Components/whiteallstars.gif");
```

Comment:
What happens before this loads in?
Is there a sharp `pop` where the background changes from white to this gif?

If so, setting the `background-color` to black should improve it, because then its only the starts popping in over and already black background.

Reply:
done :)